src/style.css

```diff
@@ -1 +1,77 @@
-/* Styles go here */
+@import url(https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap);
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0 auto;
+}
+
+.container {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  height: 60px;
+}
+
+.logo {
+  display: flex;
+  justify-content: space-between;
+  padding-right: 50px;
+}
+
+.logo-link {
+  width: 40px;
+  height: 40px;
+}
+
+.for-nav {
+  padding: 0;
+}
+
+.nav {
```

Comment:
Second point in the [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)